.circleci/config.yml

```diff
@@ -181,6 +198,44 @@ jobs:
       - main_pipeline:
           bump: prerelease
           environment: none
+          rebase_with_master: true
+
+  notify_about_merge_to_master:
```

Comment:
this is a new job that will be triggered only on master and thus will be executed on every commit to master (every merged PT to master)